stream-chat-android-ui-components-sample/src/demo/kotlin/io/getstream/chat/ui/sample/application/AppConfig.kt

```diff
@@ -19,12 +19,27 @@ package io.getstream.chat.ui.sample.application
 import io.getstream.chat.ui.sample.data.user.SampleUser
 
 object AppConfig {
-    const val apiKey: String = "qx5us2v6xvmh"
+    // const val apiKey: String = "qx5us2v6xvmh"
```

Comment:
We need to revert this before merging

Reply:
@kanat @TGazica again - could we just revert this right away and then provide a patch in the PR instead?